include/dylib.hpp

```diff
@@ -31,16 +31,25 @@
 class dylib {
 public:
 #if defined(_WIN32) || defined(_WIN64)
+    static constexpr const char *prefix = "";
     static constexpr const char *extension = ".dll";
```

Comment:
If you want to use "extension", you don't specify the dot. I suggest making it "suffix".